.github/workflows/interface_tests.yml

```diff
@@ -0,0 +1,23 @@
+name: Interface Tests
+
+on:
+  push:
+    branches: [main]
+    paths:
+      - src/interfaces/coral_web/**
+  pull_request: {}
+
+jobs:
+  interface_tests:
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: src/interfaces/coral_web
+
+steps:
+  - name: Checkout repo
+    uses: actions/checkout@v3
+  - name: Check coral_web is formatted
+    run: pnpm format
```

Comment:
before we can use `pnpm` do we need to install it or does it just work :o

Reply:
added, good call 👍